tools/qt-builder/README.md

```diff
@@ -10,311 +10,314 @@ Note that there are patches.
+Copy the files *zlib1.dll*  and *zlib.pdb* from *qt5-build\qtbase\bin* to *qt5-install\bin*.
+FIXME: Why aren't these zlib files automatically included?
 
 
 #### Uploading
 
 Create a Gzip tar file called qt5-install-5.15.2-windows.tar.gz from the qt5-install folder.
 
-Using 7-Zip:  
-* `cd` to the *qt5* folder.  
-* `7z a -ttar qt5-install-5.15.2-windows.tar qt5-install`  
+Using 7-Zip:
+* `cd` to the *qt5* folder.
+* `7z a -ttar qt5-install-5.15.2-windows.tar qt5-install`
 * `7z a -tgzip qt5-install-5.15.2-windows.tar.gz qt5-install-5.15.2-windows.tar`
 
 Upload qt5-install-5.15.2-windows.tar.gz to the Amazon S3 vircadia-public bucket, under the dependencies/vckpg directory.
-Update hifi_vcpkg.py to use this new URL. Additionally, you should make a small change to any file in the vircadia/cmake/ports 
+Update hifi_vcpkg.py to use this new URL. Additionally, you should make a small change to any file in the vircadia/cmake/ports
 directory to force the re-download of the qt-install.tar.gz during the build process for Vircadia.
 
 #### Preparing Symbols
 
-Using Python 3, Run the following command in the *qt5* folder, substituting in paths as required. The 
-*prepare-windows-symbols-for-backtrace.py* file is in the Vircadia repository's *tools\qt-builder* folder.  
-`<path>python <path>prepare-windows-symbols-for-backtrace.py qt5-install` 
+Using Python 3, Run the following command in the *qt5* folder, substituting in paths as required. The
+*prepare-windows-symbols-for-backtrace.py* file is in the Vircadia repository's *tools\qt-builder* folder.
+`<path>python <path>prepare-windows-symbols-for-backtrace.py qt5-install`
 
-This scans the qt5-install directory for dlls and pdbs,  and copies them to a *backtrace* directory. 
-Check that all dlls and pdbs are in the root of the directory.  
+This scans the qt5-install directory for dlls and pdbs,  and copies them to a *backtrace* directory.
+Check that all dlls and pdbs are in the root of the directory.
 
 Zip up this directory and upload it to Backtrace or other crash log handlng tool.
 
 
 ### Linux
-**TODO: Update this section for Qt 5.15.2**  
+**TODO: Update this section for Qt 5.15.2**
 
 #### Preparing source files
-`git clone --recursive git://code.qt.io/qt/qt5.git -b 5.12.3 --single-branch`  
-  
-*  Copy the **patches** folder to qt5  
-*   Apply patches to Qt  
-`cd qt5`  
-`git apply --ignore-space-change --ignore-whitespace patches/aec.patch`  
-`git apply --ignore-space-change --ignore-whitespace patches/qtscript-crash-fix.patch`  
-`cd ..`
+```bash
+git clone --recursive git://code.qt.io/qt/qt5.git -b 5.15.2 --single-branch
+```
+
+As of qt version 5.15.0, no patches are needed anymore.
+~~* Copy the **patches** folder to qt5~~
+~~* Apply patches to Qt~~
+~~`cd qt5`~~
+~~`git apply --ignore-space-change --ignore-whitespace patches/aec.patch`~~
+~~`git apply --ignore-space-change --ignore-whitespace patches/qtscript-crash-fix.patch`~~
+~~`cd ..`~~
```

Comment:
Let's just delete these lines rather than strike them out.
Also, delete the no longer used patch files, aec.patch and qtscript-crash-fix.patch.